website/src/components/Header.js

```diff
@@ -73,7 +73,7 @@ const Logo = () => (
           marginLeft: '0.5em',
         }}
       >
-        v5.x alpha
+        v5.x
```

Comment:
Can we make this just `v5`, rather than `v5.x`. Since `v5` isn't a semantic version number, it doesn't really make sense to imply that it is...